src/components/UI/ComponentSettings.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <transition :name="transition">
-    <div v-if="state.appearanceGroupOverlay && state.appearanceGroupOverlay.isOpen" id="component-settings-overlay">
+    <div v-if="state.appearanceGroupOverlay && state.appearanceGroupOverlay.isOpen" id="component-settings-overlay" ref="componentoverlay">
```

Comment:
is this correct that it's not in camel case? `componentOverlay`

Reply:
Updated to camelCase everywhere